src/components/composer/index.js

```diff
@@ -193,8 +194,8 @@ class ComposerWithData extends Component<Props, State> {
     this.setState({
       availableCommunities: communities,
       availableChannels: channels,
-      activeCommunity: community ? community.id : null,
-      activeChannel: activeChannel ? activeChannel.id : null,
+      activeCommunity: '',
```

Comment:
so this means that i have to select a community every time?
If i have already selected a single community at the left side navigation we should also pre-select this community here

Reply:
Agreed - it should be pre filtered if the user has selected a community or channel in the left sidebar.

Reply:
Sure I am on it to make the necessary changes for pre-selected community/channel.